ui/webui/resources/tools/build_webui.gni

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//tools/grit/preprocess_if_expr.gni")
+import("//ui/webui/resources/tools/build_webui.gni")
+import("//ui/webui/resources/tools/generate_grd.gni")
+
+# Allows brave a single template for each webui to add files to the single
+# set of resources for that webui. This is especially important for typescript
+# compiles and resource bundling. It also however allows static resources
+# to enter the generated GRD file and prevent having to override the C++
+# WebUI to add more another resource pack.
+template("brave_build_webui") {
+  # 1) Move any .ts files required to be part of the chromium TS compile
+  #    to the same gen path that the chromium files are moved to before the
+  #    rollup compile step.
+  #    Also tell chromium about the files so that they can be included
+  #    in generated GRDP.
+  # 2) Generate any GRDP files for static files and add to the chromium GRD
+
+  assert(defined(invoker.brave_base_dir),
+         "Need to know the root of the brave-specific source files")
+
+  this_path_ = rebase_path(".", "//")
+  print("Brave overriding webui target: $this_path_/$target_name")
```

Comment:
I believe instead of `$this_path_/$target_name` you can use `get_label_info(":$target_name", "label_no_toolchain")`

Reply:
I'll probably just remove this log before merging unless you think there's a reason to keep it

Reply:
> I'll probably just remove this log before merging unless you think there's a reason to keep it

I see. Let's remove it.